docs/filters.md

```diff
@@ -306,6 +306,11 @@ Aliases: `cn`
 
 HubTurbo automatically downloads detailed information about issues when the [`updated`](#updated) filter is specified, and then displays them within the issue cards in the form of a timeline.
 
+HubTurbo also supports `;` as a shorter form of OR that can be used to unify a disjunction of several qualifiers of the same type.
```

Comment:
Could we put this under the operators section? Since that's where we first talk about OR.

Also, instead of 'unify', maybe use 'express', as we probably shouldn't overload the word. :stuck_out_tongue: